src/main/scala/is/hail/HailContext.scala

```diff
@@ -339,8 +329,14 @@ class HailContext private(val sc: SparkContext,
 
     val ffConfig = FamFileConfig(quantPheno, delimiter, missing)
 
-    PlinkLoader(this, bed, bim, fam,
-      ffConfig, nPartitions, a2Reference)
+    val bedInputs = hadoopConf.globAllPartWithExt(Array(bed), Some(".bed")).sortBy(getPartNumber)
+    val bimInputs = hadoopConf.globAllPartWithExt(Array(bim), Some(".bim")).sortBy(getPartNumber)
+
```

Comment:
I think an additional check that there are no missing part numbers might be good. What do you think? Are there cases where someone would want to load just a subset of parts?